src/test/java/com/vaadin/testbenchexample/advanced/ScreenshotIT.java

```diff
@@ -44,7 +44,13 @@ public void testOnePlusTwo() throws Exception {
         // The output is placed in target/screenshot_errors
         calcView.calculate("1+2");
 
-        generateReferenceIfNotFound("oneplustwo");
+        String referenceId = "oneplustwo";
```

Comment:
We have to use the `referenceId` also in the assertion, otherwise the test will fail